src/Image.js

```diff
@@ -64,29 +67,28 @@ function ImageBase({
   image,
   adjustImage,
   deriveSizes,
-  style = {}, 
-  imgProps = {}, 
-  layoutClassName = undefined 
+  style = {},
+  imgProps = {},
+  layoutClassName = undefined
 }) {
   if (process.env.NODE_ENV !== 'production' && !image.asset.metadata) {
     console.error('Image asset doesn\'t have associated metadata. Did you forget to dereference the asset field (`image{..., asset->}`)?')
   }
 
   const { ref: sizeRef, size: displaySize } = useElementSize()
   const { src, srcSet } = useSrcSet({ config: sanityConfig, image, adjustImage })
-  const { width, height, size } = useDerivedSizes({
-    deriveSizes, 
+  const { width, height, sizes } = useDerivedSizes({
```

Comment:
Volgens mij zei je het al. Ik zou deze op `size` houden en dan bij `img`:

```jsx
<img
  sizes={sizes || `${size}px`}
  ...
```

Reply:
Scheelt een hoop wijzigingen en de kosten voor het berekenen van `size` zijn laag genoeg om het te doen terwijl je het niet gebruikt.

Reply:
Yes, dat is inderdaad zo. Bij deze een seintje